drivers/si1133/include/si1133_internals.h

```diff
@@ -0,0 +1,198 @@
+#define SI1133_PARAM_ADCSENS3     (0x0f)
+#define SI1133_PARAM_ADCPOST3     (0x10)
+#define SI1133_PARAM_MEASCONFIG3  (0x11)
+#define SI1133_PARAM_ADCCONFIG4   (0x12)
+#define SI1133_PARAM_ADCSENS4     (0x13)
+#define SI1133_PARAM_ADCPOST4     (0x14)
+#define SI1133_PARAM_MEASCONFIG4  (0x15)
+#define SI1133_PARAM_ADCCONFIG5   (0x16)
+#define SI1133_PARAM_ADCSENS5     (0x17)
+#define SI1133_PARAM_ADCPOST5     (0x18)
+#define SI1133_PARAM_MEASCONFIG5  (0x19)
+#define SI1133_PARAM_MEASRATE_H   (0x1a)
+#define SI1133_PARAM_MEASRATE_L   (0x1b)
+#define SI1133_PARAM_MEASCOUNT0   (0x1c)
+#define SI1133_PARAM_MEASCOUNT1   (0x1d)
+#define SI1133_PARAM_MEASCOUNT2   (0x1e)
+#define SI1133_PARAM_THRESHOLD0_H (0x25)
+#define SI1133_PARAM_THRESHOLD0_L (0x26)
+#define SI1133_PARAM_THRESHOLD1_H (0x27)
+#define SI1133_PARAM_THRESHOLD1_L (0x28)
+#define SI1133_PARAM_THRESHOLD2_H (0x29)
+#define SI1133_PARAM_THRESHOLD2_L (0x2a)
+#define SI1133_PARAM_BURST        (0x2b)
+/** @} */
+
+/**
+ * @name Si1133 RESPONSE0 register constants
+ * @{
+ */
+#define SI1133_RESP0_COUNTER_MASK (0x0f)
+#define SI1133_RESP0_CMD_ERR_MASK (0x10)
+#define SI1133_RESP0_SLEEP_MASK   (0x20)
+#define SI1133_RESP0_SUSPEND_MASK (0x40)
+#define SI1133_RESP0_RUNNING_MASK (0x80)
+
+/* Possible error values if SI1133_RESP0_CMD_ERR_MASK is set. */
+#define SI1133_RESP0_ERR_INVALID_COMMAND    (0x01)
+#define SI1133_RESP0_ERR_INVALID_PARAM_ADDR (0x80)
+#define SI1133_RESP0_ERR_ADC_OVERFLOW       (0x88)
+#define SI1133_RESP0_ERR_BUFFER_OVERFLOW    (0x89)
+/** @} */
+
+/**
+ * @name Si1133 Channel configuration constants
+ * @{
+ */
+#define SI1133_ADCCONFIG_DECIM_RATE_MASK    (0x60)
+#define SI1133_ADCCONFIG_DECIM_RATE_SHIFT   (5u)
+#define SI1133_ADCCONFIG_ADCMUX_MASK        (0x1f)
+#define SI1133_ADCCONFIG_ADCMUX_SHIFT       (0u)
+
+#define SI1133_ADCSENS_HSIG_MASK            (0x80)
+#define SI1133_ADCSENS_SW_GAIN_MASK         (0x70)
+#define SI1133_ADCSENS_SW_GAIN_SHIFT        (4u)
+#define SI1133_ADCSENS_HW_GAIN_MASK         (0x0f)
+#define SI1133_ADCSENS_HW_GAIN_SHIFT        (0u)
+
+#define SI1133_ADCPOST_24BIT_OUT_MASK       (0x40)
+#define SI1133_ADCPOST_POSTSHIFT_MASK       (0x38)
+#define SI1133_ADCPOST_POSTSHIFT_SHIFT      (3u)
+#define SI1133_ADCPOST_THRESH_SEL_MASK      (0x03)
+#define SI1133_ADCPOST_THRESH_SEL_SHIFT     (0u)
+
+#define SI1133_MEASCONFIG_COUNTER_IDX_MASK  (0xc0)
+#define SI1133_MEASCONFIG_COUNTER_IDX_SHIFT (6u)
+/** @} */
+
+/**
+ * @name Si1133 photodiode selection values for ADCMUX field.
+ * @{
+ */
+#define SI1133_ADCMUX_SMALL_IR    (0u)
+#define SI1133_ADCMUX_MEDIUM_IR   (1u)
+#define SI1133_ADCMUX_LARGE_IR    (2u)
+#define SI1133_ADCMUX_WHITE       (11u)
+#define SI1133_ADCMUX_LARGE_WHITE (13u)
+#define SI1133_ADCMUX_UV          (24u)
+#define SI1133_ADCMUX_DEEP_UV     (25u)
+/** @} */
+
+/**
+ * @name Si1133 constants
+ * @{
+ */
+#define SI1133_ID                              (0x33)
+#define SI1133_STARTUP_TIME                    (25000ul)    /* 25ms */
```

Comment:
```suggestion
#define SI1133_STARTUP_TIME_MS                 (25u)
```

I think it's always helpful to encode the unit with the define so it's always clear what we are dealing with.
Also no need for µs precision here.

Reply:
Done.